### PR TITLE
Add common error message in pitfalls.asciidoc

### PR DESCRIPTION
**Pull request by FruitFly638**
Add the error message `unable to upgrade ws to command server` in the Pitfalls debug developer mode section
Related ticket: https://progress.opensuse.org/issues/97952

### CODE REVIEW

File: docs/Pitfalls.asciidoc [hunk 1]
```diff
@@ -82,7 +82,7 @@ load_templates scripts accordingly.
 
 [id="debugdevelmode"]
 == Steps to debug developer mode setup
-This is basically a checklist to go through in case the developer mode is broken in your setup:
+This is basically a checklist to go through in case the developer mode is broken in your setup (e.g. you are getting the error message `unable to upgrade ws to command server`):
```
Comment (okurz) [inline docs/Pitfalls.asciidoc:85]: IMHO the text itself is fine and at the right location. I simply suggest to wrap the lines at 80 characters

**Review by okurz — CHANGES_REQUESTED** [review]
The static check that fails is about the git commit message subject line. Better phrase the git commit message as "Add common error message …" rather than "added common error message …"

Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

File: docs/Pitfalls.asciidoc [hunk 2]
```diff
@@ -82,7 +82,8 @@ load_templates scripts accordingly.
 
 [id="debugdevelmode"]
 == Steps to debug developer mode setup
-This is basically a checklist to go through in case the developer mode is broken in your setup (e.g. you are getting the error message `unable to upgrade ws to command server`):
+This is basically a checklist to go through in case the developer mode is broken in your setup
+(e.g. you are getting the error message `unable to upgrade ws to command server`):
```
Comment (FruitFly638) [inline docs/Pitfalls.asciidoc:86]: Is this better?

**Review by Martchus — COMMENTED** [review]
The 2nd commit should be squashed into the 1st one and the actual commit message should be adjusted (and not just the PR title). You can do that locally and then push your changes using the `-f`-flag.

Btw, you could have avoid the squashing by using `git commit --amend` to incorporate your latest changes into the last commit.

File: docs/Pitfalls.asciidoc (hunk 2 above)
Comment (okurz) [reply docs/Pitfalls.asciidoc:86]: Yes, thx

**Review by okurz — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**